src/components/modals/Community/Community.tsx

```diff
@@ -1,23 +1,26 @@
 import { People } from "@mui/icons-material";
 import { Alert, Badge, Box, Divider, Tooltip, Typography, useTheme } from "@mui/material";
-import React, { ReactNode } from "react";
+import React, { ReactNode, useState } from "react";
 
 import CommunityUserCard from "@/components/modals/Community/CommunityUserCard";
+import ConfirmationDialog from "@/components/utils/ConfirmationDialog";
 import { useCommunity } from "@/lib/hooks/useCommunity";
 import { ChainProfile } from "@/types/chain";
-import { CommunityUser, UserRelationship } from "@/types/community";
+import { CommunityUser, UserRelationship, UserRelationshipAction } from "@/types/community";
 
 const CommunityUserList = ({
     title,
     defaultText,
     communityUsers,
     chainProfiles,
+    onRemoveFriend,
```

Comment:
Could this be optional?

Reply:
Indeed 🖖🏻 https://github.com/mathiazom/rezervo-web/pull/117/commits/f0df7a9a16f660d08a5084343b99f854421e36bc